File: Application/Services/Operations/Finances/Dtos/YearlyExpenses/YearlyFixedExpenseTrackingDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Authentication.Dtos;
using Application.Services.Operations.Finances.Dtos.Enums;
using Application.Services.Operations.Finances.Dtos.Inheritance;
using Application.Services.Operations.Main.Companies.Dtos;
using Domain.Entities.Main.Companies;

namespace Application.Services.Operations.Finances.Dtos
{
    public class YearlyFixedExpenseTrackingDto : BaseExpenseTrackingDto
    {
        public int YearlyFixedExpenseId { get; set; }
        public YearlyFixedExpenseDto YearlyFixedExpense { get; set; }
        public DateTime Start { get; set; }
        public DateTime Expiration { get; set; }
    }
}

File: Views/src/components/financial/components/yearly-fixed-expenses-trancking/dto/yearly-fixed-expense-tracking-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { BankAccountDto } from "../../bank-account-cards/dto/bank-account-dto";
import { CardDto } from "../../bank-account-cards/dto/card-dto";
import { PixDto } from "../../bank-account-cards/dto/pix-dto";
import { YearlyFixedExpenseDto } from "../../yearly-fixed-expenses/dto/yearly-fixed-expense-dto";

export class YearlyFixedExpenseTrackingDto {
  id: number;
  companyId: number;
  company: CompanyDto;
  userId: number;
  user: MyUser;
  yearlyFixedExpenseId: number;
  yearlyFixedExpense: YearlyFixedExpenseDto;
  bankAccountId: number;
  bankAccount: BankAccountDto;
  pix: PixDto;
  pixId: number;
  othersPaymentMethods:string;
  card: CardDto;
  cardId: number;
  wasPaid: Date;
  start: Date
  expiration: Date;
  registered: Date = new Date();
  price: number;
  interest: number;
  deleted: boolean;
  // description: string;
}
